reactor-core/src/test/java/reactor/core/publisher/SinksTest.java

```diff
@@ -0,0 +1,721 @@
+/*
+ * Copyright (c) 2011-Present VMware Inc. or its affiliates, All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package reactor.core.publisher;
+
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
+import java.util.concurrent.TimeUnit;
+
+import org.awaitility.Awaitility;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+import reactor.test.StepVerifier;
+import reactor.test.subscriber.AssertSubscriber;
+import reactor.util.concurrent.Queues;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatCode;
+
+/**
+ * @author Simon Baslé
+ */
+class SinksTest {
+
+	@Nested
+	class MulticastNoWarmup {
+
+		private Sinks.StandaloneFluxSink<Integer> sink;
+		private Flux<Integer> flux;
+
+		@BeforeEach
+		void createInstance() {
+			sink = Sinks.multicastNoWarmup();
+			flux = sink.asFlux();
+		}
+
+		@Test
+		void fluxViewReturnsSameInstance() {
+			assertThat(flux).isSameAs(sink.asFlux());
+		}
+
+		@Test
+		void isAcceptingMoreThatOneSubscriber() {
+			assertThatCode(() -> {
+				flux.subscribe();
+				flux.subscribe();
+			}).doesNotThrowAnyException();
+		}
+
+		@Test
+		void honorsMultipleSubscribersBackpressure()
+				throws InterruptedException, ExecutionException {
+			ExecutorService es = Executors.newFixedThreadPool(2);
```

Comment:
Should be destroyed after the test to avoid dangling `Thread`s

Reply:
done in the dynamic test version